### PR TITLE
enabling extensions

### PR DESCRIPTION
**Pull request by jdcanas**
Merging this now that the enableWelder flag is functioning. 

I, the developer opening this PR, do solemnly pinky swear that:

- [ ] I've documented my API changes in Swagger

In all cases:

- [ ] Get a thumbsworth of review and PO signoff if necessary
- [ ] Verify all tests go green
- [ ] Squash and merge; you can delete your branch after this
- [ ] Test this change deployed correctly and works on dev environment after deployment

### CODE REVIEW

**Review by rtitle — APPROVED** [review]

**Review by Qi77Qi — APPROVED** [review]
:shipit:

File: src/main/resources/jupyter/edit-mode.js [hunk 1]
```diff
@@ -401,7 +401,5 @@ define(() => {
         location.reload(true)
     }
 
-    return {
-        load_ipython_extension: init
-    }
+    init()
```
Comment (calbach) [inline src/main/resources/jupyter/edit-mode.js:404]: Interesting  - can anyone explain how these files are executed? I tried to trace this in `init-actions.sh` but couldn't figure it out. These don't seem to have the normal Jupyter JS extension setup.

**Review by calbach — APPROVED** [review]

**Review by jmthibault79 — COMMENTED** [review]
Looks OK but I have the same question as CH

File: src/main/resources/jupyter/edit-mode.js (hunk 1 above)
Comment (jdcanas) [reply src/main/resources/jupyter/edit-mode.js:404]: ```extension_entry.js``` imports the extensions. Where it is invoked, I'm not sure. In ```init-actions.sh``` it indicates the fact it is renamed to custom.js is important, which tells me something downstream imports it, but I haven't had any luck finding what beyond that. I followed how ```google_sign_in.js``` is handled. @rtitle may know more

Comment (rtitle) [reply src/main/resources/jupyter/edit-mode.js:404]: Yeah https://github.com/DataBiosphere/leonardo/blob/develop/src/main/resources/jupyter/extension_entry.js is the entrypoint to these internal extensions. It is injected into Jupyter via the `custom.js` hook.

Note these are different than user-supplied extensions, which are installed by `jupyter nbextension install ...`.

**Pull request merged**